neuralcompression/ext/__init__.py

```diff
@@ -3,4 +3,15 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-from ._pmf_to_quantized_cdf import pmf_to_quantized_cdf
+from pathlib import Path
+
+from torch.utils.cpp_extension import load
+
+print(str(Path(__file__).resolve().parent / "pmf_to_quantized_cdf_py.cc"))
```

Comment:
Nit: factorize of `Path(__file__).resolve().parent` into `extension_folder` and reuse below. If you plan to load more extensions like this, you might create a function `load_extension` :

```
def load_extension(extension_name: str, file_names: List[str]):
   extension_folder = Path(__file__).resolve().parent
   return load(
     name=extension_name,
     sources=[str(Path(__file__).resolve().parent / file_name) for file_name in file_names],
     verbose=True,
   )
```

Reply:
Okay let me try it :).

Reply:
@QuentinDuval I updated this to try to follow your suggested format. Does it fit what you were thinking?

Reply:
Lovely.

Reply:
@QuentinDuval I am more than happy about @mmuckley's change but I am also open to other ideas for managing extensions.

Reply:
I think the main question I would be curious to have an answer for is "on what platform does the CppExtension work" and what to do to make it work on all of them. I know that torchvision does have a lot of extensions, but for instance, I never tried to install it on windows.

Reply:
My current solution follows this [thread](https://github.com/zhanghang1989/PyTorch-Encoding/issues/167). Based on this I think it will fail on Windows. The nice thing is with this new implementation we localize it to only this function.

The better thing later would be to figure out a multi-platform distribution framework and move this back to `setup.py` or to remove the C++ code.

Reply:
@mmuckley Frankly, in retrospect, it was overkill on my behalf. But, in my defense, I didn't have a strong grasp of what did and did not work with `@torch.jit.script` at the time (e.g., a Python implementation should be easily compilable).

Reply:
@0x00b1 I'm open to any implementation in any format! My only requirement is that if we have C++ code we pay the cost of distributing it.